### PR TITLE
Style personalization tab

### PR DESCRIPTION
**Pull request by alicehuanggg**

### CODE REVIEW

**Comment by alicehuanggg** [comment]
simulator screen shot - iphone 8 - 2018-07-24 at 17 49 12
simulator screen shot - iphone 8 - 2018-07-24 at 17 49 13

File: VouchrSdkShowCase-iOS/Personalization Section/Collection View Cells/MerchantCustomizationCollectionViewCell.m [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #import "CheckboxView.h"
 
 #define CHECKBOX_HEIGHT 100
+#define NUM_OF_CONFIGS 2
```
Comment (rbui) [inline VouchrSdkShowCase-iOS/Personalization Section/Collection View Cells/MerchantCustomizationCollectionViewCell.m]: wtf is this

File: VouchrSdkShowCase-iOS/Personalization Section/Collection View Cells/NoteCustomizationCollectionViewCell.m [hunk 2]
```diff
@@ -63,31 +64,33 @@ - (void)setupWithPersonalizationConfig:(PersonalizationConfigModel *)config {
     self.titleLabel.text = config.name;
 }
 
+- (void)setupCollectionViewTheme {
+    self.colorsCollectionView.backgroundColor = [UIColor colorWithRed:169.0/255.0 green:169.0/255.0 blue:169.0/255.0 alpha:1.0];
```
Comment (rbui) [inline VouchrSdkShowCase-iOS/Personalization Section/Collection View Cells/NoteCustomizationCollectionViewCell.m]: make variables for the colors like:
```
UIColor *burgandyColor = [UIColor colorWithRed:169.0/255.0 green:169.0/255.0 blue:169.0/255.0 alpha:1.0];
self.colorsCollectionView.backgroundColor = burgandyColor;
```
makes it easier to update in future

File: VouchrSdkShowCase-iOS/Personalization Section/Collection View Cells/MerchantCustomizationCollectionViewCell.m (hunk 1 above)
Comment (alicehuanggg) [reply VouchrSdkShowCase-iOS/Personalization Section/Collection View Cells/MerchantCustomizationCollectionViewCell.m]: wait no i need this

Comment (alicehuanggg) [reply VouchrSdkShowCase-iOS/Personalization Section/Collection View Cells/MerchantCustomizationCollectionViewCell.m]: or else the height of cell gets messed up, its too small

**Comment by alicehuanggg** [comment]
theme launch button
<br />
simulator screen shot - iphone 8 - 2018-07-24 at 20 26 48

**Pull request merged**